tffm/models.py

```diff
@@ -25,13 +25,14 @@ class TFFMClassifier(TFFMBaseModel):
     """
 
     def __init__(self, **init_params):
-        assert 'loss_function' not in init_params
-        init_params['loss_function'] = loss_logistic
+        if 'loss_function' not in init_params:
+            init_params['loss_function'] = loss_logistic
         self.init_basemodel(**init_params)
 
     def preprocess_target(self, y_):
         # suppose input {0, 1}, but internally will use {-1, 1} labels instead
-        assert(set(y_) == set([0, 1]))
+        if not (set(y_) == set([0, 1])):
```

Comment:
Is there a reason to do check this way? 
I'm fully OK with error message, it's great. But why not to put it in `assert` instead of separate `if` and `raise`?

Reply:
I find `AssertionError` to be ambiguous, I prefer to raise an error that explains the problem. That's a style issue though, if you prefer assertions that's OK too.